### PR TITLE
wxGUI: displaying infobar focused on first-time user (part 2)

### PR DESCRIPTION
**Pull request by lindakladivova**
After creating a new location when a user is in demolocation, second Info Bar appears offering a user to import data using Import vector data or Import raster data buttons.

A the same time, when creating a new Location, GRASS switches to the Location that a user created to its PERMANENT mapset.

### CODE REVIEW

**Comment by lindakladivova** [comment]
Info Bar 2:
infobar2

Import vector data dialog:
infobar2_1

GRASS after vector data import:
infobar_3

File: gui/wxpython/datacatalog/tree.py [hunk 1]
```diff
@@ -994,10 +1002,15 @@ def CreateLocation(self, grassdb_node):
...
+            self.SwitchMapset(grassdatabase, location, mapset)
+            location_created = True
+        return (location_created, is_user_in_demolocation)
 
     def OnCreateLocation(self, event):
         """Create new location"""
-        self.CreateLocation(self.selected_grassdb[0])
+        location_created, is_user_in_demolocation = (
+            self.CreateLocation(self.selected_grassdb[0])
+        )
```
Comment (wenzeslaus) [inline gui/wxpython/datacatalog/tree.py:1013]: There is no need to assign the variables if you are not using them, but more importantly, this should be solved in a different way as context menu on the database node does not trigger the info bar. It also makes the two functions in tree and catalog diverge too much (two different non-trivial functions which should be the same and are not).

What do you think @petrasovaa @landam, leave for another PR or resolve now?

**Review by wenzeslaus — CHANGES_REQUESTED** [review]

File: gui/wxpython/datacatalog/infomanager.py [hunk 2]
```diff
@@ -43,5 +43,19 @@ def ShowDataStructureInfo(self, onCreateLocationHandler):
...
         self.infoBar.ShowMessage(message, wx.ICON_INFORMATION, buttons)
 
+    def ShowImportDataInfo(self, OnImportOgrLayersHandler, OnImportGdalLayersHandler):
+        """Show info about the data import focused on the first-time user"""
+        buttons = [("Import vector data", OnImportOgrLayersHandler),
+                   ("Import raster data", OnImportGdalLayersHandler)]
+        message = _(
+            "You have successfully created a new Location {loc}. "
+            "Currently you are in its PERMANENT Mapset which is used for "
+            "storing your base maps to make it readily avaliable in other "
```
[the remark this answers is not in the document]
Comment (lindakladivova) [reply gui/wxpython/datacatalog/infomanager.py]: Solved.

File: gui/wxpython/datacatalog/catalog.py [hunk 3]
```diff
@@ -57,6 +57,8 @@ def __init__(self, parent, giface=None, id=wx.ID_ANY,
         # infobar manager for data catalog
         self.infoManager = DataCatalogInfoManager(infobar=self.infoBar,
                                                   giface=self.giface)
+        self.tree.showImportDataInfo.connect(self.showImportDataInfo())
```
Comment (wenzeslaus) [inline gui/wxpython/datacatalog/catalog.py:60]: You are calling the `self.showImportDataInfo()` function here, you just need to pass it to connect just like with `ShowImportDataInfo` or at line 52.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
I like these changes much better than the previous ones. The code is more straightforward and all decisions are kept in tree where is the main functionality now.

File: gui/wxpython/datacatalog/catalog.py (hunk 3 above)
Comment (lindakladivova) [reply gui/wxpython/datacatalog/catalog.py]: Has been corrected, now works as expected.

**Review by wenzeslaus — APPROVED** [review]

**Pull request merged**